.github/workflows/tox-pytest.yml

```diff
@@ -4,11 +4,92 @@ name: tox-pytest
 on: [push, pull_request]
 
 env:
-  PUDL_OUTPUT: ~/pudl-work/output
-  PUDL_INPUT: ~/pudl-work/data/
+  PUDL_OUTPUT: /home/runner/pudl-work/output
+  PUDL_INPUT: /home/runner/pudl-work/data/
 
 jobs:
-  ci-test:
+  ci-static:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        tox-env:
+          - linters
+          - docs
```

Comment:
Could we easily add the `docs` build to the list of actions which generates coverage? There are some functions in PUDL that use our metadata to build documentation pages that are only called when the docs are built.

Reply:
Yeah, definitely. Let me get the coverage working At All and then it should be pretty easy to upload artifacts. We could in theory upload artifacts for all steps, and the `linters` step would just.... not cover anything.